source/brailleDisplayDrivers/hims.py

```diff
@@ -250,7 +250,7 @@ def registerAutomaticDetection(cls, driverRegistrar: bdDetect.DriverRegistrar):
 		# Hid device
 		driverRegistrar.addUsbDevices(bdDetect.DeviceType.HID, {
 			"VID_045E&PID_940A",  # Braille Edge3S 40
-		})
+		}, True)

```

Comment:
Can you split each parameter up into new lines e.g.

```py
driverRegistrar.addUsbDevices(
	bdDetect.DeviceType.HID,
	{
		"VID_045E&PID_940A",  # Braille Edge3S 40
	},
	True
)

Reply:
I fixed it. Thanks.